web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1118,6 +1126,7 @@ fixmystreet.display = {
                 $sideReport.find('#key-tool-problems-nearby').addClass('js-back-to-report-list');
             }
             fixmystreet.set_up.map_sidebar_key_tools();
+            fixmystreet.set_up.ward_select_multiple();
```

Comment:
This won't do anything, you're switching to a individual report page that lacks this link.
The main page is then hidden, so you don't need to reinstate it when you go back either, it looks like.